x.toml

```diff
@@ -10,6 +8,23 @@ smoke-test = { path = "smoke-test" }
 toolchain = "nightly-2020-07-08"
 flags = "-Zfeatures=all"
 
+[cargo.sccache]
+bucket = "ci-artifacts.libra.org"
+prefix = "sccache/libra/"
+public = true
+envs = [
+   #To debug sccache uncomment the two lines below.
+   ["SCCACHE_ERROR_LOG","/tmp/sccache_log"],
+   ["SCCACHE_LOG", "sccache::compiler::compiler=trace"],
```

Comment:
I'll comment these two lines before merge.   But, soo useful.....

Reply:
Oh this is nice :)